### ISSUE TITLE
Failed to play PornHub links

### ISSUE THREAD

**Issue by AADI0009** [DuncteBot/skybot-lavalink-plugin#23]
## Description
Unable to play pornhub tracks :c

## Version info
**Client used:** Shoukaku

Output of `java -version`:
```
openjdk 17.0.8.1 2023-08-24
```

Logs 
```
0|java  | 2023-09-08 18:49:58.730  INFO 80238 --- [ XNIO-1 task-12] lavalink.server.io.RequestLoggingFilter  : PATCH /v3/sessions/sznbgprt3qyi4fzm/players/989607016219676693?noReplace=false, client=69.69.69.69:420, payload={"encodedTrack":"QAAAuQIAQE1PU1QgSU5URU5TRSBzaGFraW5nLCBzcXVpcnRpbmcsIHF1aXZlcmluZyBvcmdhc21zIC0gQ09NUElMQVRJT04ACGJpYml2aWRpAAAAAAAHhcgAD3BoNjNiMzVjMzlhZDA2NgABAD5odHRwczovL3d3dy5wb3JuaHViLmNvbS92aWV3X3ZpZGVvLnBocD92aWV3a2V5PXBoNjNiMzVjMzlhZDA2NgAHcG9ybmh1YgAAAAAAAAAA"}
0|java  | 2023-09-08 18:49:58.914 ERROR 80238 --- [k-1-thread-2665] c.s.d.l.t.p.LocalAudioTrackExecutor      : Error in playback of ph63b35c39ad066
0|java  | com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Media var has changed, please contact developer
0|java  |       at com.dunctebot.sourcemanagers.pornhub.PornHubAudioTrack.scoupMediaVar(PornHubAudioTrack.java:114) ~[skybot-lavalink-plugin-1.4.2.jar!/:na]
0|java  |       at com.dunctebot.sourcemanagers.pornhub.PornHubAudioTrack.loadFromMediaInfo(PornHubAudioTrack.java:90) ~[skybot-lavalink-plugin-1.4.2.jar!/:na]
0|java  |       at com.dunctebot.sourcemanagers.pornhub.PornHubAudioTrack.getPlaybackUrl(PornHubAudioTrack.java:51) ~[skybot-lavalink-plugin-1.4.2.jar!/:na]
0|java  |       at com.dunctebot.sourcemanagers.Mp3Track.loadStream(Mp3Track.java:54) ~[skybot-lavalink-plugin-1.4.2.jar!/:na]
0|java  |       at com.dunctebot.sourcemanagers.Mp3Track.process(Mp3Track.java:49) ~[skybot-lavalink-plugin-1.4.2.jar!/:na]
0|java  |       at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:104) ~[lavaplayer-1.5.0.jar!/:na]
0|java  |       at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:346) ~[lavaplayer-1.5.0.jar!/:na]
0|java  |       at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[na:na]
0|java  |       at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[na:na]
0|java  |       at java.base/java.lang.Thread.run(Thread.java:833) ~[na:na]
0|java  |       Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation: 
0|java  |   lavaplayer.version: 1.5.0
0|java  |   os.arch: amd64
0|java  |   os.name: Linux
0|java  |   os.version: 5.4.0-29-generic
0|java  |   java.vendor: Private Build
0|java  |   java.version: 17.0.8.1
0|java  |   java.runtime.version: 17.0.8.1+1-Ubuntu-0ubuntu120.04
0|java  |   java.vm.version: 17.0.8.1+1-Ubuntu-0ubuntu120.04
```

**Comment by duncte123** [comment]
> Media var has changed, please contact developer

*sigh* Yeah that happens from time to time, will fix when I find out how it works this time

**Comment by duncte123** [comment]
Fixed in version `1.6.3`, please keep in mind that this is on a new maven repository: `https://maven.lavalink.dev/releases`
